cmd/codegen/generator/generator.go

```diff
@@ -24,17 +24,18 @@ const (
 )
 
 type Config struct {
-	// Language supported by this codegen infra.
+	// Lang is the language supported by this codegen infra.
 	Lang SDKLang
 
-	// Destination directory for generated code.
+	// OutputDir is the path to place generated code.
 	OutputDir string
 
-	// Name of the module to generate code for
-	ModuleName        string
-	ModuleContextPath string
+	// ModuleName is the module name to generate code for.
+	ModuleName string
+	// ModulePath is the subpath where a module can be found.
+	ModulePath string
```

Comment:
I'd prefer to keep the name more specific and include `Context` since that's what it is. Otherwise there's 3 dirs this could be referring to (context is the whole git repo the module lives in, source is where the dagger.json lives, source subpath is where the actual source code for the module is)

Reply:
Fixed, and restored the name to `ModuleContextPath`.